core/src/main/java/de/danoeh/antennapod/core/preferences/UserPreferences.java

```diff
@@ -373,7 +373,9 @@ public static void setFollowQueue(boolean value) {
         prefs.edit().putBoolean(UserPreferences.PREF_FOLLOW_QUEUE, value).apply();
     }
 
-    public static boolean shouldSkipKeepEpisode() { return prefs.getBoolean(PREF_SKIP_KEEPS_EPISODE, true); }
+    public static boolean shouldSkipKeepEpisode() {
+        return prefs.getBoolean(PREF_SKIP_KEEPS_EPISODE, true);
+    }
```

Comment:
Please do not reformat unrelated files. It is a lot harder to use things like `git blame` when the commit that last touched a line did not actually modify the meaning. Also, this makes the PR a lot more compact.